signals/sender/sender_contains_mail_ru.yml

```diff
@@ -0,0 +1,4 @@
+name: "Sender: Root domain is mail.ru"
+type: "query"
+source: |
+  sender.email.domain.root_domain == "corp.mail.ru"
```

Comment:
```suggestion
  sender.email.domain.root_domain == "mail.ru"
```

corp.mail.ru would be the domain.domain. 
![image](https://github.com/sublime-security/sublime-rules/assets/6809735/630e6937-7495-49f7-b381-ec79389361e6)